app/models/piece.rb

```diff
@@ -18,6 +18,27 @@ def move!(x,y)
     end
   end
 
+  #Inital Capture Logic:
+  #Need game model to test id of game
+
+  #Set initial captured status to false
+  def default_captured_status
+    self.captured = false if self.captured = nil
+  end
+
+  #Find the target piece in the database base on the x,y coordinates
+  def capturable?(x,y)
+    Piece.where(:x_position => x, :y_position => y, :captured => false).where.not(:color => color).present?
+  end
+
+  #When the target is captured, change it's attributes and remove it from the board
+  def capture!(x,y)
+    opponent_piece = Piece.where(:x_position => x, :y_position => y)
```

Comment:
This will need to be associated with game object, but also take a look at `find_by` vs `where`.  This code as it stands will return an activerecord association object instead of the piece itself.  You would need append a `take` or `first` method to it to actually return an object, but you can avoid that by using the `find_by` which expects to return a specific record.